dotnet/Firely.Server.MessageSender/CommandProcessor.cs

```diff
@@ -32,6 +34,16 @@ public static RetrievePlanItem BuildRetrievePlanItem(IEnumerable<string> args)
 
         return storePlanItem;
     }
+    public static List<StorePlanItem>? BuildStorePlanItems(string command, IEnumerable<string> args)
+    {
+        var storePlanItems = command switch
+        {
+            "dir" => CreateResourcesFromDirectory(args),
```

Comment:
Consider having an addition parameter: "d" "c" "u" "ups" so as to delete, create or update all resources in the directory. If you do so, you would have to refactor `CreateResourcesFromDirectory` to reuse the file import part.

Reply:
Importing from a directory is no longer interactive. I changed the ExecuteStorePlan to use Upsert. This is currently a little strange because PubSub always requires a `versionId` and it does not seem possible to generate these, but anyway I made a separate ticket about that.